packages/malloy/src/malloy.ts

```diff
@@ -726,19 +734,10 @@ export class Model implements Taggable {
    * @param queryName Name of the query to retrieve.
    * @return A prepared query.
    */
-  public getPreparedQueryByName(
-    queryName: string,
-    options?: PrepareQueryOptions
-  ): PreparedQuery {
+  public getPreparedQueryByName(queryName: string): PreparedQuery {
     const query = this.modelDef.contents[queryName];
     if (query?.type === 'query') {
-      return new PreparedQuery(
-        query,
-        this.modelDef,
-        this.problems,
-        options?.replaceMaterializedReferences ?? false,
-        queryName
-      );
+      return new PreparedQuery(query, this.modelDef, this.problems, queryName);
```

Comment:
Calling `getPreparedQuery` is "leaving the runtime world," and thus loses any runtime-specific settings, like `replaceMaterializedReferences`.